runtime/binding-asyncapi/pom.xml

```diff
@@ -256,38 +241,15 @@
                   </fileMappers>
                 </artifactItem>
               </artifactItems>
-              <includes>io/aklivity/zilla/specs/binding/asyncapi/schema/asyncapi.schema.patch.json,
-                io/aklivity/zilla/specs/binding/mqtt/kafka/schema/mqtt.kafka.schema.patch.json,
-                io/aklivity/zilla/specs/binding/http/schema/http.schema.patch.json,
-                io/aklivity/zilla/specs/binding/sse/schema/sse.schema.patch.json,
-                io/aklivity/zilla/specs/binding/kafka/schema/kafka.schema.patch.json,
-                io/aklivity/zilla/specs/binding/tcp/schema/tcp.schema.patch.json,
-                io/aklivity/zilla/specs/binding/tls/schema/tls.schema.patch.json,
+              <includes>
+                io/aklivity/zilla/specs/binding/asyncapi/schema/asyncapi.schema.patch.json,
                 io/aklivity/zilla/specs/binding/asyncapi/schema/asyncapi.2.6.0.schema.json,
                 io/aklivity/zilla/specs/binding/asyncapi/schema/asyncapi.3.0.1.schema.json,
                 io/aklivity/zilla/specs/binding/asyncapi/schema/asyncapi.3.0.2-zilla.schema.json
```

Comment:
Please remove this include as we don't use our own Asyncapi scheme (also it was removed).